src/argilla/utils/dependency.py

```diff
@@ -63,22 +63,29 @@ def _compare_versions(
         )
 
 
-def require_version(requirement: str, func_name: Optional[str] = None) -> None:
+def require_version(requirement: Union[str, List[str]], func_name: Optional[str] = None) -> None:
```

Comment:
Maybe this can be renamed from `require_version` to `require_dependency` or `require_dependencies`

Reply:
I will let `require_dependencies` like the renamed decorator